SCClassLibrary/Common/GUI/Base/QWindow.sc

```diff
@@ -103,7 +103,16 @@ Window {
 		resizable = true,
 		border = true,
 		deprecatedServerArgument,
-		scroll = false;
+		scroll = false,
+		checkForExistingWindow = false;
```

Comment:
renaming suggestion: `reuseExisting = false`. This makes clear what is done when it exists.

Reply:
thanks, this is indeed much better